### PR TITLE
ini_file - added note in documentation for utf-8 bom

### PR DESCRIPTION
**Pull request by russoz**
##### SUMMARY
Adding docs as an aftermath of #2578 .

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
ini_file

### CODE REVIEW

**Comment by jpmens** [comment]
shipit

**Review by aminvakil — APPROVED** [review]
shipit

File: plugins/modules/files/ini_file.py [hunk 1]
```diff
@@ -79,6 +79,7 @@
 notes:
    - While it is possible to add an I(option) without specifying a I(value), this makes no sense.
    - As of Ansible 2.3, the I(dest) option has been changed to I(path) as default, but I(dest) still works as well.
+   - As of version 3.2.0 of the community.general collection UTF-8 BOM markers are discarded in files.
```
Comment (felixfontein) [inline plugins/modules/files/ini_file.py:82]:
```suggestion
   - As of community.general 3.2.0, UTF-8 BOM markers are discarded when reading files.
```

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@russoz thanks for improving!
@jpmens @aminvakil thanks for reviewing!

**Pull request merged**